theano/tensor/opt.py

```diff
@@ -3360,6 +3360,25 @@ def local_incsubtensor_of_zeros(node):
             return
 
 
+@register_canonicalize
+@register_stabilize
```

Comment:
This isn't a stabilization opt. Is should be in register_specialize instead of register_stabilize